### DISCUSSION TITLE
Port issue with Spare Block Chain 9444?

### DISCUSSION THREAD

**Discussion by PC101guy** [Chia-Network/chia-blockchain#7121] [Support]
It appears to be some form of port issue on windows 10 running Sparecoin and silicoin as they both use these ports.   One will not run while the other is running.   I am not sure on how to fix.

**Comment by loppefaaret** [comment]
why don't you go ask them ? this is the chia-blockchain repo

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
